Source/g_game.c

```diff
@@ -1819,6 +1819,14 @@ static void G_DoLoadGame(void)
   char vcheck[VERSIONSIZE];
   byte saveg_complevel = MBFVERSION;
 
+  // [crispy] loaded game must always be single player.
+  // Needed for ability to use a further game loading, as well as
+  // cheat codes and other single player only specifics.
+  // if (startloadgame == -1)
```

Comment:
I guess the `startloadgame` check was there so that it is still possible to load a network game from the command line? I really don't remember anymore...

Reply:
Yep:
https://github.com/fabiangreffrath/crispy-doom/commit/501051f5cbafdc792ddad1f994f48d7bb391d21f

Reply:
Welcome back! Yes, but why I can't load multiplayer saves from the menu? I want to change that. I actually use them in local co-op games.